### PR TITLE
Fix helm list limitation issue for CF

### PR DESCRIPTION
**Pull request by zhangtbj**

### CODE REVIEW

File: make/upgrade [hunk 1]
```diff
@@ -23,7 +23,7 @@ get_uaa_secret () {
 }
 
 # Argument? (**)
-RELEASE=$(helm list --namespace "${NAMESPACE}" -q | tail -n 1)
+RELEASE=$(helm list list -d -r --max 1 --namespace "${NAMESPACE}" -q | tail -n 1)
```
Comment (mook-as) [inline make/upgrade]: That's too many `list`s

File: make/run [hunk 2]
```diff
@@ -9,7 +9,7 @@ UAA_NAMESPACE="uaa"
 CA_CERT=internal-ca-cert
 
 # Determine the kube revision of the chart controlling the specified namespace.
-release_version() { helm list --namespace "$1" | awk '{ print $2 }' | tail -n 1 ; }
+release_version() { helm list -d -r --max 1 --namespace "$1" | awk '{ print $2 }' | tail -n 1 ; }
```
Comment (mook-as) [inline make/run]: Consider using the full argument names (`--date`, `--reverse`) in scripts.

**Comment by zhangtbj** [comment]
Hi @mook-as Sorry for the error, I have refined related code.

Pls review.

**Comment by zhangtbj** [comment]
I also fix the parameter name in uaa-fissile-release repo:
https://github.com/SUSE/uaa-fissile-release/pull/82

**Comment by zhangtbj** [comment]
Hi @mook-as , Sure thing. I have bumped uaa release to the latest. Pls review.

**Comment by mook-as** [comment]
Thanks! Merged.

**Pull request merged**